src/main/java/software/sigma/internship/service/AnswerServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package software.sigma.internship.service;
```

Comment:
It would be better to create impl package and put all implementations to this package